### PR TITLE
Release

### PR DESCRIPTION
**Pull request by henrytao-me**

### CODE REVIEW

File: packages/web-workers/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
```
Comment (henrytao-me) [inline packages/web-workers/CHANGELOG.md:10]: Is this a correct place? Should I create new block for current release version and date? cc @lemonmade

Comment (lemonmade) [reply packages/web-workers/CHANGELOG.md:10]: Yes, please create a block for the new version and date

File: packages/web-workers/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remote-ui/web-workers",
-  "version": "1.2.9",
+  "version": "1.2.10",
```
Comment (lemonmade) [inline packages/web-workers/package.json:3]: There is technically a new feature in here so IMO this should be `1.3.0` instead

Comment (henrytao-me) [reply packages/web-workers/package.json]: Fixed!

**Review by lemonmade — APPROVED** [review]

**Pull request merged**